integration-test/src/test/scala/io/phdata/clients/RoleClientIntegrationSpec.scala

```diff
@@ -0,0 +1,210 @@
+package io.phdata.clients
+
+import java.util.UUID
+
+import cats.effect.IO
+import io.phdata.itest.fixtures.{HiveTest, IntegrationTest, _}
+import io.phdata.models.{Manager, ReadOnly, ReadWrite}
+import io.phdata.services.UGILoginContextProvider
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import cats.implicits._
+
+class RoleClientIntegrationSpec extends FlatSpec with Matchers with HiveTest with IntegrationTest with BeforeAndAfterAll{
```

Comment:
Are all tests in this class passing? Can you note with a comment any that are not?

Reply:
No not all are passing. I will add a note on which aren't. There is a hive ranger command grid. 
https://cwiki.apache.org/confluence/display/RANGER/Hive+Commands+to+Ranger+Permission+Mapping